engine/crates/parser-openapi/src/graph/mod.rs

```diff
@@ -299,6 +299,10 @@ pub enum HttpMethod {
     Put,
     Delete,
     Patch,
+    Head,
+    Trace,
```

Comment:
didn't know it even existed

Reply:
Nor did I.  Still not sure what it's used for